tools/test_utils/src/MakeMeshes.cpp

```diff
@@ -53,11 +55,11 @@ ReadLegacyMeshFile(std::filesystem::path const& file_path)
     }
 
     std::size_t num_edges;
-    nc_inq_dim(ncidp, dimid, read_name.get(), &num_edges);
-    std::shared_ptr<double> node_x(new double[num_nodes]);
-    std::shared_ptr<double> node_y(new double[num_nodes]);
-    std::shared_ptr<int> edge_nodes(new int[num_edges * 2]);
-    std::shared_ptr<int> edge_type(new int[num_edges]);
+    nc_inq_dim(ncidp, dimid, read_name.data(), &num_edges);
+    std::shared_ptr<double> node_x(new double[num_nodes], std::default_delete<double[]>());
```

Comment:
Did we change all those unique_ptrs to vectors in the release/v2.0.0 branch? If yes, then never mind. When the release branch is merged, this will be taken care off. Otherwise Please make the change here.

Reply:
I think that I changed them all in the release branch